### PR TITLE
feat(logging): track expiration in error message

### PR DESCRIPTION
**Pull request by arein**
# Description

We're currently seeing issues for the Flutter SDK when trying to connect. We're unsure if this is because of a misconfigured device or a bug in the SDK.

## Due Diligence

* [ ] Breaking change
* [ ] Requires a documentation update
* [ ] Requires a e2e/integration test update

### CODE REVIEW

**Review by arein — COMMENTED** [review]
sorry half of this is fmt / clippy

File: relay_rpc/src/jwt.rs [hunk 1]
```diff
@@ -202,11 +202,11 @@ pub trait VerifyableClaims: Serialize + DeserializeOwned {
         let now = Utc::now().timestamp();
 
         if matches!(basic.exp, Some(exp) if now - time_leeway > exp) {
-            return Err(JwtError::Expired);
+            return Err(JwtError::Expired { expiration: basic.exp } );
         }
 
         if now + time_leeway < basic.iat {
-            return Err(JwtError::NotYetValid);
+            return Err(JwtError::NotYetValid { basic_iat: basic.iat, now_time_leeway: now + time_leeway });
```
Comment (chris13524) [inline relay_rpc/src/jwt.rs]: Is it worth logging `time_leeway` separately from `now`? Comment above says `time_leeway` is configurable

File: relay_rpc/src/jwt.rs [hunk 2]
```diff
@@ -202,11 +209,16 @@ pub trait VerifyableClaims: Serialize + DeserializeOwned {
...
+            return Err(JwtError::Expired {
+                expiration: basic.exp,
+            });
         }
 
         if now + time_leeway < basic.iat {
-            return Err(JwtError::NotYetValid);
+            return Err(JwtError::NotYetValid {
+                basic_iat: basic.iat,
+                now_time_leeway: now + time_leeway,
```
Comment (chris13524) [inline relay_rpc/src/jwt.rs:220]: Is it worth logging `time_leeway` separately from `now? Comment above says `time_leeway` is configurable

**Review by chris13524 — APPROVED** [review]
LGTM

**Pull request merged**